docs/kubernetes/operator/instrumenting-python.md

```diff
@@ -0,0 +1,179 @@
+- `musl` based containers instrumentation requires an [extra annotation](https://opentelemetry.io/docs/kubernetes/operator/automatic/#annotations-python-musl) and operator v0.113.0+.
+- To enable logs auto-instrumentation, refer to [auto-instrument python logs](https://opentelemetry.io/docs/kubernetes/operator/automatic/#auto-instrumenting-python-logs).
+- To disable specific instrumentation libraries, refer to [excluding auto-instrumentation](https://opentelemetry.io/docs/kubernetes/operator/automatic/#python-excluding-auto-instrumentation).
+- For a full list of configuration options, refer to [Python specific configuration](https://opentelemetry.io/docs/zero-code/python/configuration/#python-specific-configuration).
+- For Python specific limitations when using the OpenTelemetry operator, refer to [Python-specific topics](https://opentelemetry.io/docs/zero-code/python/operator/#python-specific-topics).
+
+
+## Guided example to instrument a Python app with EDOT Python SDK on Kubernetes
+
+In the following example you will learn how to:
+
+- Deploy an example Python app in a dedicated namespace.
+- Enable auto-instrumentation of the application following any of the supported methods, such as:
+  - Adding an annotation to the namespace.
+  - Adding an annotation to the deployment pods.
+- Verify that auto-instrumentation libraries are injected and configured correctly.
+- Confirm data is flowing to **Kibana Observability**.
+
+Before continuing, ensure you have performed the [installation of the operator](./README.md), and confirm that the following `Instrumentation` object exists in the system:
+
+```bash
+$ kubectl get instrumentation -n opentelemetry-operator-system
+NAME                      AGE    ENDPOINT                                                                                                
+elastic-instrumentation   107s   http://opentelemetry-kube-stack-daemon-collector.opentelemetry-operator-system.svc.cluster.local:4318
+```
+
+Example auto-instrumentation steps:
+
+1. Create a `python` namespace and run a deployment named `python-app`
+
+    ```bash
+    # Python Namespace and Deployment
+    kubectl create namespace python
+    kubectl apply -f - <<EOF
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app: python-app
+      name: python-app
+      namespace: python
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: python-app
+      template:
+        metadata:
+          name: python-app
+          labels:
+            app: python-app
+        spec:
+          containers:
+            - image: andrewgizas/python-otel-auto:latest
+              imagePullPolicy: Always
+              name: python-app
+              env: 
+              - name: OTEL_LOG_LEVEL
+                value: "debug"
+              - name: AGENT_HAS_STARTED_IF_YOU_SEE
+                value: "Exception while exporting metrics HTTPConnectionPool"
```

Comment:
Not sure we need this

Reply:
> The reported output is supposed to be an example that might not match exactly the user's outcome. Do you think I should clarify that in the doc to avoid confusion?

If there's a collector running you shouldn't see this though?